gensim/test/test_utils.py

```diff
@@ -259,6 +263,46 @@ def test_save_as_line_sentence_ru(self):
             self.assertEqual(sentences, ref_sentences)
 
 
+class HashTest(unittest.TestCase):
+    def setUp(self):
+        self.expected = {
+            u'команда': 1725507386,
```

Comment:
add English here (works same for broken & current impl) + Japanese maybe

Reply:
OK, done